Port number must be set on quarkus.spring-cloud-config.url to pass the test

## Describe the bug
My Quarkus project reads its configuration from a central Spring Cloud Config Server. I've set the corresponding properties, so it works when running `mvn quarkus:dev` but fails when running the tests `mvn clean package`

### Expected behavior
Such command should work. It runs the test using the  quarkus.spring-cloud-config.url property without the need to add a special port.
quarkus.spring-cloud-config.url=http://my-spring-config-service

### Actual behavior
Got an exception:
`Caused by: java.lang.RuntimeException: Failed to start quarkus
Caused by: java.lang.RuntimeException: Unable to obtain configuration from Spring Cloud Config Server at http://my-spring-config-service
Caused by: java.lang.IllegalArgumentException: port p must be in range 0 <= p <= 65535`

To fix this, I added the port number
quarkus.spring-cloud-config.url=http://my-spring-config-service:80


## To Reproduce
Create any Hello World application with spring-cloud-config extension, configure the URL targeting SCCS without a port number
then run 
`mvn clean package`

### Output of `uname -a` or `ver`
Microsoft Windows [version 10.0.19041.804]
### Output of `java -version`
openjdk version "11" 2018-09-25
### GraalVM version (if different from Java)
openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment GraalVM CE 21.1.0 (build 11.0.11+8-jvmci-21.1-b05)
OpenJDK 64-Bit Server VM GraalVM CE 21.1.0 (build 11.0.11+8-jvmci-21.1-b05, mixed mode, sharing)
### Quarkus version or git rev
1.13.4.Final
### Build tool (ie. output of `mvnw --version` or `gradlew --version`)
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)

--- a/extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/VertxSpringCloudConfigGateway.java
+++ b/extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/VertxSpringCloudConfigGateway.java
@@ -179,8 +179,8 @@ public class VertxSpringCloudConfigGateway implements SpringCloudConfigClientGat
         final String requestURI = finalURI(applicationName, profile);
         String finalURI = getFinalURI(applicationName, profile);
         HttpRequest<Buffer> request = webClient
-                .get(baseURI.getPort(), baseURI.getHost(), requestURI)
-                .ssl(baseURI.getScheme().contains("https") ? true : false)
+                .get(getPort(baseURI), baseURI.getHost(), requestURI)
+                .ssl(isHttps(baseURI))
                 .putHeader("Accept", "application/json");
         if (springCloudConfigClientConfig.usernameAndPasswordSet()) {
             request.basicAuthentication(springCloudConfigClientConfig.username.get(),
@@ -206,6 +206,14 @@ public class VertxSpringCloudConfigGateway implements SpringCloudConfigClientGat
                 }
             }
         });
+    }
+
+    private boolean isHttps(URI uri) {
+        return uri.getScheme().contains("https");
+    }
+
+    private int getPort(URI uri) {
+        return uri.getPort() != -1 ? uri.getPort() : (isHttps(uri) ? 443 : 80);
     }
 
     private String getFinalURI(String applicationName, String profile) {

